cmd/mailbox-controller/main.go

```diff
@@ -95,54 +91,59 @@ func main() {
 		}
 	}()
 
-	// create edgeSharedInformerFactory
-	espwRestConfig, err := espwClientOpts.ToRESTConfig()
+	// create space-aware client
```

Comment:
The package name suggests "multi-space client" more than "space-aware client". I think "multi-space client" is the better term. "space-aware client" says to me that this is like an ordinary client extended with space awareness. For me, "multi-space client" gives more emphasis to the "multi" part. The client being constructed is not like a regular kube client at all, it is a different sort of client exclusively and specifically focused on navigating spaces.